firmware/sys/at_client/Kconfig

```diff
@@ -1,3 +1,13 @@
 menu "AT Client"
-# TO DO
+    config BAUDRATE
+    int "BAUDRATE DEFAULT"
+    default 115200
```

Comment:
The baudrate I think that sould be selective, no free asigning, example i could express a baudrate of 9325. when the acceptable baudrates are `9600`, `14400` `115200`. To more reference follow the standards baud-rate acceptable https://www.ni.com/es-cr/support/documentation/supplemental/18/serial-baud-rates-supported-by-ni-visa.html#section-118519396

Reply:
Also, I think that it's used an specify board, the kconfig don´t get the unsupported baud-rates as a choice